setup.py

```diff
@@ -49,12 +33,11 @@ def readme():
       long_description=readme(),
       license='Apache License 2.0',
       author="Luke Campbell",
-      author_email="lcampbell@asascience.com",
+      author_email="luke.campbell@gdit.com",
```

Comment:
@Bobfrat you should probably change this to your email. I am just a contributor now, not the maintainer.